project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
 logLevel := Level.Warn
 
-addSbtPlugin("io.gatling" % "gatling-sbt" % "2.2.0")
-addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "4.0.0")
+addSbtPlugin("io.gatling" % "gatling-sbt" % "3.0.0")
+//addSbtPlugin("com.typesafe.sbteclipse" % "sbteclipse-plugin" % "4.0.0")
```

Comment:
why?

Reply:
this should be added in your personal plugin (under `.sbt/1.0/plugins/plugins.sbt`)
I will remove this line in a following fixup

Reply:
removed